src/test/java/org/opensearch/neuralsearch/highlight/SemanticHighlighterTests.java

```diff
@@ -67,6 +69,11 @@ public void setUp() throws Exception {
 
         fieldType = new TextFieldMapper.TextFieldType(TEST_FIELD);
 
+        // Initialize EventStatsManager for tests
+        NeuralSearchSettingsAccessor mockSettingsAccessor = mock(NeuralSearchSettingsAccessor.class);
+        when(mockSettingsAccessor.isStatsEnabled()).thenReturn(true);
+        EventStatsManager.instance().initialize(mockSettingsAccessor);
```

Comment:
You can use this static test helper I made here: https://github.com/opensearch-project/neural-search/blob/main/src/testFixtures/java/org/opensearch/neuralsearch/util/TestUtils.java#L402-L407